src/persist/src/postgres.rs

```diff
@@ -239,10 +238,13 @@ impl PostgresConsensus {
         //
         // See: https://github.com/MaterializeInc/materialize/issues/13975
         // See: https://www.cockroachlabs.com/docs/stable/configure-zone.html#variables
-        tx.batch_execute("ALTER TABLE consensus CONFIGURE ZONE USING gc.ttlseconds = 600;")
-            .await?;
-
+        tx.batch_execute(&format!(
```

Comment:
can we just run `batch_execute` on the client instead? maybe we don't even need the transaction

Reply:
I can clean that up!

Reply:
it's possible that this would improve contention even further, so might want to do it before we spend too much time looking at the new performance